test/src/unit-capi-query_2.cc

```diff
@@ -3131,3 +3131,73 @@ TEST_CASE_METHOD(
 
   remove_array(array_name);
 }
+
+TEST_CASE_METHOD(
+    Query2Fx,
+    "C API: Test subarray, sparse, set point ranges list",
+    "[capi][query_2][sparse][ranges-list]") {
+  SECTION("- No serialization") {
+    serialize_ = false;
+  }
+  SECTION("- Serialization") {
+    serialize_ = true;
+  }
+  std::string array_name = "subarray_sparse_default_bulk_ranges";
+  remove_array(array_name);
+  const uint64_t local_DIM_DOMAIN[4] = {1, 12, 1, 12};
+  create_sparse_array(array_name, local_DIM_DOMAIN);
+
+  // Allocate subarray with an unopened array
+  tiledb_array_t* array;
+  int rc = tiledb_array_alloc(ctx_, array_name.c_str(), &array);
+  CHECK(rc == TILEDB_OK);
+
+  // Open array
+  rc = tiledb_array_open(ctx_, array, TILEDB_READ);
+  CHECK(rc == TILEDB_OK);
+
+  // Create query
+  tiledb_query_t* query = nullptr;
+  rc = tiledb_query_alloc(ctx_, array, TILEDB_READ, &query);
+  CHECK(rc == TILEDB_OK);
+  rc = tiledb_query_set_layout(ctx_, query, TILEDB_UNORDERED);
+  CHECK(rc == TILEDB_OK);
+
+  // Add ranges
+  // coalesces, having only 3 ranges...
```

Comment:
What does this comment mean? Also suggest to remove the commented line below if unnecessary.

Reply:
comment outdated, removed, related to the no longer needed dataset in that next commented line.